src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -602,6 +600,70 @@ ad_utility::streams::stream_generator ExportQueryExecutionTrees::
   co_yield "\n</sparql>";
 }
 
+// _____________________________________________________________________________
+template <>
+ad_utility::streams::stream_generator ExportQueryExecutionTrees::
+    selectQueryResultToStream<ad_utility::MediaType::sparqlJson>(
+        const QueryExecutionTree& qet,
+        const parsedQuery::SelectClause& selectClause,
+        LimitOffsetClause limitAndOffset,
+        CancellationHandle cancellationHandle) {
+  // This call triggers the possibly expensive computation of the query result
+  // unless the result is already cached.
+  std::shared_ptr<const Result> result = qet.getResult();
+  result->logResultSize();
+  LOG(DEBUG) << "Converting result IDs to their corresponding strings ..."
+             << std::endl;
+  auto selectedColumnIndices =
+      qet.selectedVariablesToColumnIndices(selectClause, false);
+
+  const auto& idTable = result->idTable();
+
+  auto vars = selectClause.getSelectedVariablesAsStrings();
+  std::ranges::for_each(vars, [](std::string& var) { var = var.substr(1); });
+  co_yield absl::StrCat("{\"head\":{\"vars\":[\"", absl::StrJoin(vars, "\",\""),
+                        "\"]},\"results\":{\"bindings\":[");
```

Comment:
To be a little bit safer here,
create an `nlohmann::json` with the variables, and then use its `dump()` to fill the `head`.
(Maybe we have odd variable names etc, we want the exporter to be robust).